reana_workflow_controller/rest.py

```diff
@@ -1861,3 +1995,91 @@ def _mark_workflow_as_deleted_in_db(workflow):
     current_db_sessions = Session.object_session(workflow)
     current_db_sessions.add(workflow)
     current_db_sessions.commit()
+
+
+def get_specification_diff(workflow_a, workflow_b, output_format=None):
+    """Return differences between two workflow specifications.
+
+    :param workflow_a: The first workflow to be compared.
+    :type: reana_db.models.Workflow instance.
+    :param workflow_a: The first workflow to be compared.
+    :type: reana_db.models.Workflow instance.
+    :param output_format: Sets output format. Optional.
+    :type: String. One of ['unified', 'context', 'html'].
```

Comment:
Why do we set to `None` the default in the signature and later we say that it will be `unified` by default? If we default to `unified` we do not need `None` around, do we?